app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
I would also have thought that the code existed in Blacklight so it wasn't necessary to copy it into our local app...